test/functional-portable/corerp/noncloud/resources/recipe_terraform_test.go

```diff
@@ -181,6 +187,7 @@ func Test_TerraformRecipe_KubernetesPostgres(t *testing.T) {
 							ValidateLabels(false),
 						validation.NewK8sPodForResource(appName, "postgres").
 							ValidateLabels(false),
+						// validation.NewK8sSecretForResourceWithResourceName(secretResourceName),
```

Comment:
Is this to be deleted?

Reply:
i will uncomment this to test functional test updates once bicep PR is merged in.